sample-apps/bundles-cart-transform/extensions/cart-merge-expand/src/api.rs

```diff
@@ -0,0 +1,127 @@
+#![allow(non_camel_case_types, non_snake_case)]
```

Comment:
Do we need to have `api.rs` or the macro should create a gql file? cc @davejcameron